util/dvsim/Deploy.py

```diff
@@ -428,6 +428,7 @@ def _define_attrs(self):
             "pre_run_cmds": False,
             "run_cmd": False,
             "run_opts": False,
+            "bazel_opts": False,
```

Comment:
IMO this is not needed.

Reply:
So I believe the desire of #13430 is to enable us to run the (closed source) manufacturer specific test hooks in dvsim. to do that requires building the bazel target with some special flags (that need to be piped through to the sim.mk file (see here: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/tests/closed_source/BUILD.bazel;drc=52e3af7bac2ca09c482ea96a14fced4d8c617971;l=25). How do you propose doing so without having some "bazel_opts" to pipe through the config files?

Reply:
after discussion (which is clarified below), this is not needed because we do not need command line support for the bazel opts. will remove.

Reply:
also we discussed that `bazel_opts` should be renamed `sw_build_opts`

Reply:
Fixed.